packages/edge-config/package.json

```diff
@@ -63,6 +62,9 @@
     "tsup": "8.0.1",
     "typescript": "5.3.3"
   },
+  "peerDependencies": {
+    "@opentelemetry/api": "~1.0"
```

Comment:
This should be a devDependency too, right?

Reply:
@dferber90 no, we want this to be a peer dependency, because that should allow us to use whatever version of `@opentelemetry/api` the client uses. If we make it a dev dependency, then it might be a different version and might not be able to resolve the `getTracer` call.